dev/tests/integration/testsuite/Magento/Sales/_files/guest_quote_with_addresses.php

```diff
@@ -14,7 +14,7 @@
 $product = $objectManager->create(\Magento\Catalog\Model\Product::class);
 $product->setTypeId('simple')
     ->setAttributeSetId($product->getDefaultAttributeSetId())
-    ->setName('Simple Product')
+    ->setName('Simple Product For Guest Quote')
```

Comment:
We could not change current fixtures
It's BIC and could lead to some failures in already created tests

Reply:
> It's BIC and could lead to some failures in already created tests
Travis has passed this change. But I you recommend then I can rollback my changes here.

Reply:
> We could not change current fixtures

**fixed**

Thank you, @naydav !!